src/sales/order-status-custom.md

```diff
@@ -77,15 +77,14 @@ A status setting cannot be unassigned from a state if the status is currently in
 |Order Status|Order State||
 |--- |--- |--- |
 |Processing|processing|When the state of new orders is set to “Processing”, the _Automatically Invoice All Items_ option becomes available in the configuration.|
-|Suspected Fraud|fraud||
-|Pending Payment|pending_payment||
-|Payment Review|payment_review||
-|Suspected Fraud|fraud||
-|Pending|pending||
-|On Hold|holded||
-|Complete|complete||
-|Closed|closed||
-|Canceled|canceled||
-|PayPal Canceled Reversal|payment_canceled_reversal||
-|PayPal Reversed|paypal_reversed||
-|Pending PayPal|pending_paypal||
+|Suspected Fraud|fraud|Sometimes orders paid via PayPal, eway or any other payment gateway, mark the orders as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```

Comment:
eway is removed from 2.4 and deprecated in 2.3.

```suggestion
|Suspected Fraud|fraud|Sometimes orders paid via PayPal or another payment gateway are marked as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```

Reply:
Done